mitodl/open-edx-plugins
# feat: added mitxpro-openedx-extensions

### What are the relevant tickets?
Closes https://github.com/mitodl/hq/issues/4142

### Description (What does it do?)
Adds mitxpro-openedx-extensions in open-edx-plugins

### How can this be tested?
- Start edx-platform and mitxpro containers.
- Clone this project in the src folder present in your Openedx work directory.
- Install pants locally and run the command in the project folder: `pants package ::`. This will create a tar file `openedx-companion-auth-1.3.0.tar.gz` in the dist folder.
- Configure openedx with mitxpro using https://github.com/mitodl/mitxpro/blob/master/docs/configure_open_edx.md
- Install the tar file in your lms by first creating the lms shell using `make lms-shell` command in the devstack folder and then `pip install /edx/src/open-edx-plugins/dist/openedx-companion-auth-1.3.0.tar.gz`.
- Open a private window and go to localhost:18000. You should be redirected to mitxpro signin page.

Note: Tested this on both master and quince release
